webcompat/form.py

```diff
@@ -233,14 +233,13 @@ def build_formdata(form_object):
     # Preparing the body
     body = u"""{metadata}
 **URL**: {url}
-**Browser / Version**: {browser}
-**Operating System**: {os}
-**Tested Another Browser**: {browser_test_type}
-
 **Problem type**: {problem_type}
 **Description**: {description}
-**Steps to Reproduce** {steps_reproduce}
 
+**Steps to Reproduce**: {steps_reproduce}
```

Comment:
I'd prefer if we had Steps to Reproduce as the last item, since it can be the longest... (also this undoes https://github.com/webcompat/webcompat.com/commit/168338448c71e012a7d3d90081d6e3b6dd7e61a7, which just landed. Maybe not intentionally?). So let's put browser/version, os, tested another browser back above problem type please.

So we want something like this:

```
....
**Problem type**: {problem_type}
**Description**: {description}
**Steps to Reproduce**:
{steps_reproduce}
""".

(With `{steps_reproduce}` on a new line)

Thanks!